### DISCUSSION TITLE
Using runner class

### DISCUSSION THREAD

**Discussion by polyted** [open-mmlab/mmcv#2512] [General]
**Checklist**

1. I have searched related issues but cannot get the expected help. Yes
2. I have read the FAQ documentation but cannot get the expected help. Yes

I notice the mmcv __init__.py file has these comments:

The following modules are not imported to this level, so mmcv may be used
without PyTorch.
- runner
- parallel
- op
- device

What should I do to enable importing and usage of runner in my code? For example I'd like to run something like this without import errors:

from mmcv.runner import get_dist_info

Thank you.

**Comment by zhouzaida** [comment]
Hi, if you installed PyTorch, importing `get_dist_info` from `mmcv.runner` should be OK.

**Comment by zhouzaida** [comment]
This topic is suitable for discussion in [the block](https://github.com/open-mmlab/mmcv/discussions). I converted it into a discussion and welcome to have more discussions.

**Comment by zixian6406** [comment]
Using 'from mmengine.dist import get_dist_info' to replace
